app/global-translations/locale-en.json

```diff
@@ -213,6 +230,9 @@
   "label.anchor.editpermissions": "Edit Permissions",
   "#Inputs": "..",
   "label.input.name": "Name",
+  "label.input.unittype": "Unit Type",
+	"label.input.baseprice": "Base Price",
```

Comment:
why extra space here?

Reply:
Pushed. But this is not getting updated
![Screenshot from 2021-09-10 21-11-04](https://user-images.githubusercontent.com/43112139/132880358-9aac4865-ede5-4f09-84d8-fb7ed958bdda.png)

Reply:
Done